pkg/registry/core/pod/etcd/eviction.go

```diff
@@ -83,7 +84,16 @@ func (r *EvictionREST) Create(ctx api.Context, obj runtime.Object) (runtime.Obje
 
 		// If it was false already, or if it becomes false during the course of our retries,
 		// raise an error marked as a 429.
-		ok, err := r.checkAndDecrement(pod.Namespace, pod.Name, pdb)
+		var ok bool
+		err = retry.RetryOnConflict(retry.DefaultBackoff, func() error {
```

Comment:
What is the maximum duration this could take? If it's more than the global timeout (currently 1 minute) bad things will happen.